src/AccessibilityInsights.sln

```diff
@@ -471,9 +471,7 @@ Global
 		{7A7C04EA-8279-4D69-959D-E804DD81BC70}.Release|x86.Build.0 = Release|Any CPU
 		{7A7C04EA-8279-4D69-959D-E804DD81BC70}.SignedRelease|Any CPU.ActiveCfg = Release|Any CPU
 		{7A7C04EA-8279-4D69-959D-E804DD81BC70}.SignedRelease|x64.ActiveCfg = Release|Any CPU
```

Comment:
Should these lines also go away?

Reply:
Discussed offline--VS auto generates this and will restore these lines as well as the undesired silently when the SignedRelease configuration is selected